### PR TITLE
Fix solution and project to load and build in VS 2019

### PR DESCRIPTION
**Pull request by tombuMS**

### CODE REVIEW

File: MREUnityRuntime/MREUnityRuntime.sln [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27703.2035
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30204.135
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MREUnityRuntimeLib", "MREUnityRuntimeLib\MREUnityRuntimeLib.csproj", "{534F3A2C-9DC5-4E0A-9AEB-7945D939B1B7}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MREUntiyRuntime", "MREUnityRuntimeLib\MREUnityRuntimeLib.csproj", "{4D8EA76B-B56F-405E-96BD-15E15B0F3A6A}"
```
Comment (stevenvergenz) [inline MREUnityRuntime/MREUnityRuntime.sln:6]: Misspelled

File: MREUnityRuntime/MREUnityRuntime.sln [hunk 2]
```diff
@@ -12,22 +12,19 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug_Android|Any CPU = Debug_Android|Any CPU
```
Comment (stevenvergenz) [inline MREUnityRuntime/MREUnityRuntime.sln:15]: Does this mean we're ditching the Debug_Android build target?

Comment (tombuMS) [reply MREUnityRuntime/MREUnityRuntime.sln:15]: Eric already removed all cases of debug logging for Android when he brought in Brent's changes.  He did not replace before leaving the project.  I am removing this until we have a decision on a better way to handle debug builds.

File: MREUnityRuntime/MREUnityRuntime.sln (hunk 1 above)
Comment (tombuMS) [reply MREUnityRuntime/MREUnityRuntime.sln]: Fixed

**Pull request merged**